tests/ArraySchemaValidatorTest.php

```diff
@@ -78,7 +78,7 @@ public function testCoerceArrayValidString(): void {
     expect($validator->isValid())->toBeTrue();
     $validated = $validator->getValidatedInput();
 
-    expect($validated)->toBeSame(['coerce_array' => $input]);
+    expect($validated)->toBeSame(shape('coerce_array' => $input));
```

Comment:
`$validated` is of a shape type. Comparing it to an array type using `===` is a bit strange.